src/viam/components/power_sensor/power_sensor.py

```diff
@@ -68,15 +68,14 @@ async def get_power(self, *, extra: Optional[Dict[str, Any]] = None, timeout: Op
             print("The power is", power, "Watts")
 
         Returns:
-            float: power in watts
+            float: The power reading in watts.
         """
         ...
 
     async def get_readings(
         self, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs
     ) -> Mapping[str, SensorReading]:
-        """Obtain the measurements/data specific to this sensor.
-        If a sensor is not configured to have a measurement or fails to read a piece of data, it will not appear in the readings dictionary.
+        """Get the measurements or readings that this power sensor provides. If a sensor is not configured to have a measurement or fails to read a piece of data, it will not appear in the readings dictionary.
```

Comment:
```suggestion
        """Get the measurements or readings that this power sensor provides. If a sensor is not configured correctly or fails to read a piece of data, it will not appear in the readings dictionary.
```

What does "configured to have a measurement" mean? Configured by the user? If so, where? Is it possible to make this active voice?